package.json

```diff
@@ -170,7 +170,7 @@
     "resolve": "^1.20.0",
     "rollup": "^2.52.7",
     "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-polyfill-node": "^0.7.0",
+    "rollup-plugin-polyfill-node": "^0.8.0",
```

Comment:
this is only semi-related, it gets rid of the `Circular dependency: polyfill-node.global.js -> polyfill-node.global.js` warning when running `yarn build-devtools` (https://github.com/FredKSchott/rollup-plugin-polyfill-node/pull/17). We still have the circular stream polyfill warnings though.